pandas/core/window.py

```diff
@@ -0,0 +1,1339 @@
+    def aggregate(self, arg, *args, **kwargs):
+        return super(Expanding, self).aggregate(arg, *args, **kwargs)
+
+    agg = aggregate
+
+    @Substitution(name='expanding')
+    @Appender(_doc_template)
+    @Appender(_shared_docs['count'])
+    def count(self):
+        return super(Expanding, self).count()
+
+    @Substitution(name='expanding')
+    @Appender(_doc_template)
+    @Appender(_shared_docs['apply'])
+    def apply(self, func, args=(), kwargs={}):
+        return super(Expanding, self).apply(func, args=args, kwargs=kwargs)
+
+    @Substitution(name='expanding')
+    @Appender(_doc_template)
+    @Appender(_shared_docs['sum'])
+    def sum(self):
+        return super(Expanding, self).sum()
+
+    @Substitution(name='expanding')
+    @Appender(_doc_template)
+    @Appender(_shared_docs['max'])
+    def max(self, how='max'):
+        return super(Expanding, self).max(how=how)
+
+    @Substitution(name='expanding')
+    @Appender(_doc_template)
+    @Appender(_shared_docs['min'])
+    def min(self, how='min'):
+        return super(Expanding, self).min(how=how)
+
+    @Substitution(name='expanding')
+    @Appender(_doc_template)
+    @Appender(_shared_docs['mean'])
+    def mean(self):
+        return super(Expanding, self).mean()
+
+    @Substitution(name='expanding')
+    @Appender(_doc_template)
+    @Appender(_shared_docs['median'])
+    def median(self, how='median'):
+        return super(Expanding, self).median(how=how)
+
+    @Substitution(name='expanding')
+    @Appender(_doc_template)
+    @Appender(_shared_docs['std'])
+    def std(self, ddof=1):
+        return super(Expanding, self).std(ddof=ddof)
+
+    @Substitution(name='expanding')
+    @Appender(_doc_template)
+    @Appender(_shared_docs['var'])
+    def var(self, ddof=1):
+        return super(Expanding, self).var(ddof=ddof)
+
+    @Substitution(name='expanding')
+    @Appender(_doc_template)
+    @Appender(_shared_docs['skew'])
+    def skew(self):
+        return super(Expanding, self).skew()
+
+    @Substitution(name='expanding')
+    @Appender(_doc_template)
+    @Appender(_shared_docs['kurt'])
+    def kurt(self):
+        return super(Expanding, self).kurt()
+
+    @Substitution(name='expanding')
+    @Appender(_doc_template)
+    @Appender(_shared_docs['quantile'])
+    def quantile(self, quantile):
+        return super(Expanding, self).quantile(quantile=quantile)
+
+    @Substitution(name='expanding')
+    @Appender(_doc_template)
+    @Appender(_shared_docs['cov'])
+    def cov(self, other=None, pairwise=None, ddof=1):
+        return super(Expanding, self).cov(other=other, pairwise=pairwise, ddof=ddof)
+
+    @Substitution(name='expanding')
+    @Appender(_doc_template)
+    @Appender(_shared_docs['corr'])
+    def corr(self, other=None, pairwise=None):
+        return super(Expanding, self).corr(other=other, pairwise=pairwise)
+
+class EWM(_Rolling):
+    """
+    .. versionadded:: 0.18.0
+
+    Provides exponential weighted functions
```

Comment:
Can you put this on the first line (switch with versionadded), otherwise it is the versionadded that you see in the autosummary tables